packages/std/src/os/adodb.rs

```diff
@@ -126,9 +126,6 @@ impl ActionFee {
     }
 }
 
-#[cw_serde]
-pub struct MigrateMsg {}
```

Comment:
I think we probably should have kept these for future use. Should be fine for now but we should test migration on chain ASAP.

Reply:
Onchain test migration is done ([tx](https://testnet-explorer.publicawesome.dev/stargaze/tx/4E5E080E8C9B44717DD8CF62492180D75C6611A5879E6A23D0CC6586BD774E62
)).

It is tested with timelock contract as follow.
1. App is created with timelock component. ([tx](https://testnet-explorer.publicawesome.dev/stargaze/tx/0E89AB5878576B6A91D238886AD3515A6BAA702048EA25ECD291D2BB7AAC69D0))
2. Compiled the timelock contract. (For testing purpose, updated timelock's `Cargo.toml` to use `version = "1.0.1-rc.1"`
 ) 
3. Deployed the code id. (3924)
4. Migrated the contract to use code id 3924.